docs/meshstack.meshmarketplace.metering.md

```diff
@@ -63,6 +63,10 @@ meshStack interprets the `costs` property for each plan and automatically mainta
 
 The following sections detail how meshStack interprets OSB API plan [cost objects](https://github.com/openservicebrokerapi/servicebroker/blob/v2.15/profile.md#cost-object) to create meshStack metering product catalog entries.
 
+### Metrics-based Metering
+
+See [Metrics-based Metering](meshstack.meshmarketplace.metrics-metering.md) for details about how to charge your services usage-based.
```

Comment:
I think it would be nice to add a little bit more information here and perhaps an example or two on what use cases you could consider.

Reply:
extended it a bit, but all details are available in the linked chapter. But i teased that a bit more ;)